modules/azure/master/master.tf

```diff
@@ -21,7 +21,7 @@ resource "azurerm_storage_account" "tectonic_master" {
 }
 
 resource "azurerm_storage_container" "tectonic_master" {
-  name                  = "${var.cluster_name}-vhd-master"
+  name                  = "${var.cluster_name}-master"
```

Comment:
is it worth to include 'vhd' in the name so its easier to distinguish that this is a disk?